### ISSUE TITLE
Fondue outputs cause errors when running metadata tabulate

### ISSUE THREAD

**Issue by lina-kim** [bokulich-lab/q2-fondue#119]
# Issue
Though I don't generally have issues running `qiime metadata tabulate` on other QIIME outputs, outputs from `q2-fondue` consistently give me this error:

```
There was an issue with loading the file test-scrape-bioproject.qza as metadata:

  Metadata file must be encoded as UTF-8 or ASCII. The following error occurred when decoding the file:

  'utf-8' codec can't decode byte 0xb0 in position 12: invalid start byte

  There may be more errors present in the metadata file. To get a full report, sample/feature metadata files can be validated with Keemei: https://keemei.qiime2.org

  Find details on QIIME 2 metadata requirements here: https://docs.qiime2.org/2022.2/tutorials/metadata/
```

# Example 
Keemei validated the metadata TSV generated by `qiime tools export` and determined it was a valid metadata file.

Command used to generate QZA file as follows. The only publication in the `fondue-test` collection was https://doi.org/10.1038/s41467-019-13036-1.
```
qiime fondue scrape-collection \
--p-library-type user \
--p-user-id ${user-id} \
--p-api-key ${api-key} \
--p-collection-name fondue-test \
--o-bioproject-ids test-scrape-bioproject.qza \
--o-run-ids test-scrape-run.qza \
--o-other-ids test-scrape-other-ids.qza \
--verbose
```

This was run in a conda environment with QIIME 2 version 2021.8.

**Comment by adamovanja** [comment]
Thanks for opening up this issue @lina-kim. I tried to reproduce the error with the same QIIME 2 version 2021.8 and the publication you mentioned - but in all cases I do not get this error on my local machine.
Do you encounter the issue also in a new conda environment with the newest Q2 version 2022.2 installed within?

**Comment by lina-kim** [comment]
Hmm you're right, it doesn't cause this error with a 2022.2 QIIME environment. This wasn't exactly a controlled experiment, as I created a new environment with updated versions of both QIIME 2 and q2-fondue. There was also no error with an updated version of q2-fondue (as of today) and QIIME 2 version 2021.8, so I wonder if the issue was with my install of an older fondue release. Sadly I didn't think to note the fondue version I saw the issue with.

Either way, error is no more. Thanks for looking into this @adamovanja!
